src/test/java/co/rsk/bitcoinj/script/StandardRedeemScriptParserTest.java

```diff
@@ -10,156 +10,116 @@
+        Assert.assertTrue(StandardRedeemScriptParser.isStandardMultiSig(standardRedeemScriptParser.extractStandardRedeemScriptChunks()));
+    }
 
-        assertEquals(2, parser.getM());
+    @Test
+    public void isStandardMultiSig_whenEmptyScript_shouldReturnFalse() {
+        Assert.assertFalse(StandardRedeemScriptParser.isStandardMultiSig(new Script(new byte[]{}).getChunks()));
     }
 
     @Test
-    public void getM_from_multiSig_standard_redeem_script() {
-        Script redeemScript = RedeemScriptUtils.createStandardRedeemScript(btcECKeyList);
-        RedeemScriptParser parser = RedeemScriptParserFactory.get(redeemScript.getChunks());
+    public void isStandardMultiSig_whenNonStandardErpRedeemScriptChunks_shouldReturnFalse() {
+        List<ScriptChunk> nonStandardRedeemScriptChunks = RedeemScriptUtils.createErpRedeemScript(
+            defaultRedeemScriptKeys,
+            RedeemScriptUtils.getEmergencyRedeemScriptKeys(),
+            CSV_VALUE
+        ).getChunks();
+        Assert.assertFalse(StandardRedeemScriptParser.isStandardMultiSig(nonStandardRedeemScriptChunks));
+    }
 
-        assertEquals(2, parser.getM());
+    @Test
+    public void isStandardMultiSig_whenP2shRedeemScriptChunks_shouldReturnFalse() {
+        List<ScriptChunk> p2shErpRedeemScriptChunks = RedeemScriptUtils.createP2shErpRedeemScript(
+            defaultRedeemScriptKeys,
+            RedeemScriptUtils.getEmergencyRedeemScriptKeys(),
+            CSV_VALUE
+        ).getChunks();
+        Assert.assertFalse(StandardRedeemScriptParser.isStandardMultiSig(p2shErpRedeemScriptChunks));
     }
 
     @Test
-    public void extractStandardRedeemScriptChunks_whenGetScriptChunksFromStandardRedeemScriptParser_shouldReturnScriptChunks() {
-        // Arrange
-        Script redeemScript = RedeemScriptUtils.createStandardRedeemScript(btcECKeyList);
-        List<ScriptChunk> expectedRedeemScriptChunks = redeemScript.getChunks();
-        RedeemScriptParser redeemScriptParser = RedeemScriptParserFactory.get(redeemScript.getChunks());
+    public void isStandardMultiSig_whenFlyoverRedeemScriptChunks_shouldReturnFalse() {
+        Script p2shRedeemScriptChunks = RedeemScriptUtils.createP2shErpRedeemScript(
+            defaultRedeemScriptKeys,
+            RedeemScriptUtils.getEmergencyRedeemScriptKeys(),
+            CSV_VALUE
+        );
 
-        // Act
-        List<ScriptChunk> actualRedeemScripChunks = redeemScriptParser.extractStandardRedeemScriptChunks();
+        Sha256Hash derivationArgumentsHash = Sha256Hash.of(new byte[]{1});
+        List<ScriptChunk> flyoverRedeemScriptChunks = RedeemScriptUtils.createFlyoverRedeemScript(
+            derivationArgumentsHash.getBytes(), p2shRedeemScriptChunks
+        ).getChunks();
 
-        // Assert
-        assertEquals(expectedRedeemScriptChunks, actualRedeemScripChunks);
+        Assert.assertFalse(StandardRedeemScriptParser.isStandardMultiSig(flyoverRedeemScriptChunks));
```

Comment:
maybe a paremeterized test with these three cases?

Reply:
There is not JUnit 5 in this project.